src/aspire/utils/fft.py

```diff
@@ -39,12 +54,29 @@ def centered_ifft2(x):
 
 
 def centered_fft2(x):
+    """
+    Calculate a centered, two-dimensional inverse FFT
+    :param x: The two-dimensional signal to be transformed.
+        The inverse FFT is only applied along the last two dimensions.
+    :return: The centered inverse Fourier transform of x.
+    """
+
+    x = ifftshift(ifftshift(x, -2), -1)
```

Comment:
I performed the changes because it was small and I have changed all these lines in the PR anyway but... in the future it might be more efficient if unrelated issues uncovered in reviews just become their own issue/PR.

Reply:
Ok thanks. I will be more restrictive in the future.